packages/quantic/force-app/main/default/lwc/quanticCitation/quanticCitation.html

```diff
@@ -17,7 +17,7 @@
         {citationTitle}
       </p>
     </a>
-    <c-quantic-tooltip light-theme class="slds-show_medium">
+    <c-quantic-tooltip light-theme class="slds-show_medium" target={template.host}>
```

Comment:
Does it really need to be the host? can't it be like the `<li> element`?

Reply:
Yea I just was thinking maybe we instead rename that property to something like `tooltip-for` and we document it so that people can actually use it without having to know what the template.host is :)